root/static/scripts/tests/relationship-editor.js

```diff
@@ -530,14 +531,14 @@ relationshipEditorTest("MBS-5389: added recording-recording relationship appears
 relationshipEditorTest("backwardness of submitted relationships is preserved (MBS-7636)", function (t) {
     t.plan(2);
 
-    var source = {
+    const source = {
             entityType: "recording",
             gid: fakeGID0
-        },
-        target = {
+    };
```

Comment:
You fixed indentation of the `};`, but the object contents are still indented too much (8 spaces instead of 4), for both source and target

Reply:
I think this problem is in the new commits!

Reply:
*is solved in the new commits

Reply:
@monicaq21 did you push the new commits? I still don't see any here that resolve these. :)

Reply:
I think it is in my commit with the comment "fixed comma-dangle issue in relationship-editor.js"

Reply:
Yes, it is in my commit with the comment "minor fixes of code"

Reply:
It doesn't appear to resolve the two most recent comments I left though... I'll link them here just in case:
https://github.com/metabrainz/musicbrainz-server/pull/1316#discussion_r356339225
https://github.com/metabrainz/musicbrainz-server/pull/1316#discussion_r356339597

Reply:
Oh yes sorry!! I was actually looking at something else... I will fix it now.

Reply:
Thanks :) https://github.com/metabrainz/musicbrainz-server/pull/1316#discussion_r356339597 is fixed now, but still not https://github.com/metabrainz/musicbrainz-server/pull/1316#discussion_r356339225 yet